pkg/apis/core/validation/validation_test.go

```diff
@@ -7591,192 +7591,214 @@ func TestValidateContainers(t *testing.T) {
 		AllowPrivileged: true,
 	})
 
-	successCase := []core.Container{
-		{Name: "abc", Image: "image", ImagePullPolicy: "IfNotPresent", TerminationMessagePolicy: "File"},
-		// backwards compatibility to ensure containers in pod template spec do not check for this
-		{Name: "def", Image: " ", ImagePullPolicy: "IfNotPresent", TerminationMessagePolicy: "File"},
-		{Name: "ghi", Image: " some  ", ImagePullPolicy: "IfNotPresent", TerminationMessagePolicy: "File"},
-		{Name: "123", Image: "image", ImagePullPolicy: "IfNotPresent", TerminationMessagePolicy: "File"},
-		{Name: "abc-123", Image: "image", ImagePullPolicy: "IfNotPresent", TerminationMessagePolicy: "File"},
-		{
-			Name:  "life-123",
-			Image: "image",
-			Lifecycle: &core.Lifecycle{
-				PreStop: &core.LifecycleHandler{
-					Exec: &core.ExecAction{Command: []string{"ls", "-l"}},
-				},
-			},
-			ImagePullPolicy:          "IfNotPresent",
-			TerminationMessagePolicy: "File",
-		},
+	successCases := []struct {
+		title, line string
+		containers  []core.Container
+		opts        PodValidationOptions
```

Comment:
can drop this change, we don't need a new validation option for this... if the field is set, validate it